Formula/terrascan.rb

```diff
@@ -35,14 +35,14 @@ def install
     EOS
 
     expected = <<~EOS
-      \tPolicies Validated  :\t203
+      \tPolicies Validated  :\t\\d+
       \tViolated Policies   :\t0
       \tLow                 :\t0
       \tMedium              :\t0
       \tHigh                :\t0
     EOS
 
-    assert_match expected, shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws")
+    assert_match(/#{expected}/, shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws"))
```

Comment:
Can we change this to use 2 `assert_match`es instead? One that matches the text for everything except "Policies Validated" and another that uses a regex for "Policies Validated".

Reply:
Something like:
```ruby
expected = <<~EOS
  \tViolated Policies   :\t0
  \tLow                 :\t0
  \tMedium              :\t0
  \tHigh                :\t0
EOS

output = shell_output("#{bin}/terrascan scan -f #{testpath}/ami.tf -t aws")
assert_match expected, output
assert_match(/Policies Validated:\s+\d+/, output)
```

Reply:
certainly!